src/components/Combobox/Combobox.tsx

```diff
@@ -284,7 +284,7 @@ const Combobox: React.FC<ComboboxProps> & {
             value={filterValue}
             wrapperClassName={cx(styles.comboboxSearchWrapper, searchWrapperClassName)}
             className={styles.comboboxSearch}
-            inputAriaLabel="Search for content"
+            searchInputAriaLabel="Search for content"
```

Comment:
Same here. It should be `inputAriaLabel={searchInputAriaLabel}` and searchInputAriaLabel should be a prop of Combobox with default value = `Search for content`

Reply:
Fixed!